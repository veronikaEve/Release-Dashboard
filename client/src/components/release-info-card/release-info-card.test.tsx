import { render } from "@testing-library/react";
import ReleaseInfoCard from "./release-info-card";

describe("ReleaseInfoCard", () => {
  const props = {
    releaseInfo: {
      branchName: "test_branch_name",
      currentVersion: "1.1.1",
      commitHash: "1ab2c3",
      openedBy: "testName",
      lastUpdatedBy: "anotherTestName",
    },
  };

  test("should match snapshot", () => {
    const { asFragment } = render(<ReleaseInfoCard {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
