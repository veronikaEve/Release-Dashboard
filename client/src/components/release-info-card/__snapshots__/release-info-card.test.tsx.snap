// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReleaseInfoCard should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="release-info-card text-center card"
  >
    <div
      class="card-header"
    >
      Release Info
    </div>
    <div
      class="card-body"
    >
      <div
        class="info-group row"
      >
        <div
          class="info-group__label col-md-4"
        >
          Name:
        </div>
        <div
          class="info-group__info col-md-8"
        >
          Wazzaaaap
        </div>
      </div>
      <div
        class="info-group row"
      >
        <div
          class="info-group__label col-md-4"
        >
          Name:
        </div>
        <div
          class="info-group__info col-md-8"
        >
          Wazzaaaap
        </div>
      </div>
      <div
        class="info-group row"
      >
        <div
          class="info-group__label col-md-4"
        >
          Name:
        </div>
        <div
          class="info-group__info col-md-8"
        >
          Wazzaaaap
        </div>
      </div>
      <div
        class="info-group row"
      >
        <div
          class="info-group__label col-md-4"
        >
          Name:
        </div>
        <div
          class="info-group__info col-md-8"
        >
          Wazzaaaap
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
